dashboard/admin.py

```diff
@@ -9,13 +9,25 @@
 from .models import CourseViewOption, Course
 
 from django.forms.models import ModelForm
+from typing import Tuple, Any
+from typing_extensions import Protocol
+
+from django.http import HttpRequest
```

Comment:
We don't have a consistent strategy for imports, but I think it might make sense to group the Django imports here and in other places in this PR.

Reply:
Ok I'll move this. PEP-8 says

> Imports should be grouped in the following order:
> 
> Standard library imports.
> Related third party imports.
> Local application/library specific imports.
> You should put a blank line between each group of imports.

But yeah, we haven't been very consistent about grouping these

Reply:
Huh, that's kind of what I have done in the past, but it's nice to know what they recommend. Thanks for looking at that.